autonomi/src/client/archive.rs

```diff
@@ -93,13 +93,15 @@ impl Archive {
             .as_secs();
         meta.modified = now;
         self.map.insert(new_path.to_path_buf(), (data_addr, meta));
+        debug!("File renamed successfully in the archive");
```

Comment:
Could we add context here, i.e., add in the vars `old_path` and `new_path`?
```suggestion
        debug!("File renamed successfully in the archive, old path: {old_path:?} new path: {new_path:?}");
```